widgets/pulseaudio.lua

```diff
@@ -33,44 +33,49 @@ local function factory(args)
     --  '/device\\.string/ p' prints the line containing 'device.string:'.
     
     local settings    = args.settings or function()
+            widgettext  = volume_now.left .. "%"
```

Comment:
This too is just a personal settings.

You can just add the tooltip to `pulseaudio.widget` from `rc.lua`.

Anyway, you can write it in the wiki entry, in a dedicated section.